server/src/main/java/io/crate/execution/engine/indexing/ShardedRequests.java

```diff
@@ -55,11 +58,13 @@ public final class ShardedRequests<TReq extends ShardRequest<TReq, TItem>, TItem
     public ShardedRequests(Function<ShardId, TReq> requestFactory, RamAccounting ramAccounting) {
         this.requestFactory = requestFactory;
         this.ramAccounting = ramAccounting;
+        this.validatorsCache = new HashMap<>();
```

Comment:
Could be done on variable declaration.

Reply:
Done but in different place, since I moved cache creation/clearance to another place. 
See https://github.com/crate/crate/pull/14373#discussion_r1260651275